buildSrc/src/main/java/Configuration.kt

```diff
@@ -0,0 +1,70 @@
+import org.gradle.api.JavaVersion
+import org.gradle.api.Project
+import java.util.Properties
+
+const val BOM_VERSION = "2.0.3"
+const val AUTH_FOUNDATION_VERSION = "2.0.3"
+const val OAUTH2_VERSION = "2.0.3"
+const val WEB_AUTHENTICATION_UI_VERSION = "2.0.3"
+const val LEGACY_TOKEN_MIGRATION_VERSION = "2.0.3"
+const val MIN_SDK = 23
+const val COMPILE_SDK = 35
+const val TARGET_SDK = 35
+val SOURCE_COMPATIBILITY = JavaVersion.VERSION_11
+val TARGET_COMPATIBILITY = JavaVersion.VERSION_11
+val JVM_TARGET = JavaVersion.VERSION_11.toString()
+
+data class OssrhCredentials(
```

Comment:
I don't see the Ossrh class and functions in this class being used anywhere. Will that be done in a future PR?